app/javascript/packs/application.js

```diff
@@ -17,19 +17,9 @@ window.$ = $;
 RailsUJS.start();
 ActiveStorage.start();
 
-// *
-// IMPLEMENTATION NOTE 2/22/21
-// I want us to be able to import honeycrisp JS directly from honeycrisp, but the main JS we need is all included
-// right inside of the cfa_styleguide_main sprockets JS file. I'm planning on making a PR to honeycrisp to change this imminently. - SB
-//
-//import "<%#= File.join(Gem.loaded_specs['cfa-styleguide'].full_gem_path, 'app', 'assets', 'javascripts', 'honeycrisp') %>";
-
 import "../lib/honeycrisp";
 
-// *
-<% unless Rails.env.test? %>
-  AjaxMixpanelEvents.init();
-<% end %>
+AjaxMixpanelEvents.init();
```

Comment:
i had a desire to keep this behavior (don't do mixpanel ajax stuff in full-stack tests) so I added another commit to try to restore it but didn't actually test to see if it works yet

Reply:
I agree with the idea of the test environment not sending the Mixpanel AJAX messages. I tested out a different approach to accomplish the same end:

```
pageData.sendMixpanelBeacon
```

I figure it's good for the test suite to _run_ the JS, just not to actually do the XHR.

Reply:
fantastic

i have revised my added commit to just be the deletey parts